src/main/java/dev/streamx/cli/ingestion/IngestionArguments.java

```diff
@@ -12,9 +12,15 @@ public class IngestionArguments {
         description = "Address of 'rest-ingestion-service'",
         showDefaultValue = Visibility.ALWAYS,
         defaultValue = DEFAULT_INGESTION_URL)
-    void propagateIngestionUrl(String ingestionUrl) {
-      IngestionClientContext context = CDI.current().select(IngestionClientContext.class).get();
-      context.setIngestionUrl(ingestionUrl);
-    }
+  void propagateIngestionUrl(String ingestionUrl) {
+    IngestionClientContext context = CDI.current().select(IngestionClientContext.class).get();
+    context.setIngestionUrl(ingestionUrl);
   }
 
+  @Option(names = "--oauth2-bearer",
```

Comment:
Due to https://clig.dev/#arguments-and-flags section ** Do not read secrets directly from flags **
This should be made interactive or pass path to file containing oauth2-bearer